R/findOptimalThreadCount.R

```diff
@@ -18,20 +22,21 @@
 #' @examples
 #' \dontrun{
 #' # Finding the best performing thread count for each benchmarked data.table function with a data size of 1000 rows and 10 columns:
-#' optimalThreads <- findOptimalThreadCount(1000, 10)
+#' optimalThreads <- data.table.threads::findOptimalThreadCount(1000, 10)
 #' }
 
-findOptimalThreadCount <- function(rowCount, colCount) {
-
+findOptimalThreadCount <- function(rowCount, colCount, times = 10, verbose = TRUE)
+{
   setDTthreads(0)
   maxThreads <- getDTthreads()
   results <- list()
 
   for (threadCount in 1:maxThreads) {
-    results[[threadCount]] <- runBenchmarks(rowCount, colCount, threadCount)
+    results[[threadCount]] <- runBenchmarks(rowCount, colCount, threadCount, times, verbose)
   }
 
-  result.list <- do.call(rbind, results)
-  class(result.list) <- "data_table_threads_benchmark"
-  result.list
+  results.dt <- rbindlist(results)
+  results.dt <- results.dt[, .(threadCount, expr, minTime = min, maxTime = max, medianTime = median)]
```

Comment:
please use Seconds instead of Time
or just min, max, median for column names, and call the variable name `seconds.dt`